kafka-ui-react-app/src/components/Wizard/WizardForm/KafkaConnect/KafkaConnect.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useEffect, useState } from 'react';
+import * as S from 'components/Wizard/WizardForm/WizardForm.styled';
+import { Button } from 'components/common/Button/Button';
+import Input from 'components/common/Input/Input';
+import { FormError } from 'components/common/Input/Input.styled';
+import { ErrorMessage } from '@hookform/error-message';
+import { useFormContext } from 'react-hook-form';
+import PlusIcon from 'components/common/Icons/PlusIcon';
+import IconButtonWrapper from 'components/common/Icons/IconButtonWrapper';
+import CloseIcon from 'components/common/Icons/CloseIcon';
+
+type PropType = {
+  handleAddKafkaConnect: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  remove: (index: number) => void;
+  fields: TField[];
+};
+type TField = {
+  id: string;
+  name: string;
+  url: string;
+  isAuth: boolean;
+  username: string;
+  password: string;
+};
+const KafkaConnect: React.FC<PropType> = ({
+  handleAddKafkaConnect,
+  remove,
+  fields,
+}) => {
+  const [newKafkaConnect, setNewKafkaConnect] = useState(false);
+  const methods = useFormContext();
+  const connect = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```

Comment:
what is this connect means? Are you connecting to some remote server or what?

Reply:
just showing  connects, i will replace with showConnects